Item/Item.module.less

```diff
@@ -104,5 +110,24 @@
 				}
 			}
 		});
+
+		.active({
```

Comment:
hm.. do you have any reason(or requirement) why added `active` on Item component? Enact Item actually doesn't have any pressed effect on click or touch. From my UX/GUI understanding, red dot + red underline should be applied to `selected`
@Djspaceg What do you think?

Reply:
You are right. I applied it to selected. Thank you!